### PR TITLE
Update tuning guide

### PR DESCRIPTION
**Pull request by cberner**
Explain when it's appropriate to add more worker threads.

### CODE REVIEW

File: presto-docs/src/main/sphinx/admin/tuning.rst [hunk 1]
```diff
@@ -17,8 +17,10 @@ These configuration options may require tuning in specific situations:
 
 * ``task.max-worker-threads``:
   Sets the number of threads used by workers to process splits. Increasing this number
-  can improve throughput, if worker CPU utilization is low, but will cause increased
-  heap space usage.
+  can improve throughput, if worker CPU utilization is low and all the threads are in use,
```
Comment (martint) [inline presto-docs/src/main/sphinx/admin/tuning.rst]: comma between throughput and if is not needed.

File: presto-docs/src/main/sphinx/admin/tuning.rst [hunk 2]
```diff
@@ -17,8 +17,10 @@ These configuration options may require tuning in specific situations:
...
+  but will cause increased heap space usage. You can check the number of threads being
```
Comment (martint) [inline presto-docs/src/main/sphinx/admin/tuning.rst]: I'd rephrase as "The number of active threads is available via the .... JMX stat".

**Comment by cberner** [comment]
Updated

**Comment by martint** [comment]
looks good
